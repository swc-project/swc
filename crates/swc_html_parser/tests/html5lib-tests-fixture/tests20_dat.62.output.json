{
  "type": "Document",
  "span": {
    "start": 1,
    "end": 33
  },
  "mode": "quirks",
  "children": [
    {
      "type": "Element",
      "span": {
        "start": 0,
        "end": 0
      },
      "tagName": "html",
      "namespace": "http://www.w3.org/1999/xhtml",
      "attributes": [],
      "children": [
        {
          "type": "Element",
          "span": {
            "start": 0,
            "end": 0
          },
          "tagName": "head",
          "namespace": "http://www.w3.org/1999/xhtml",
          "attributes": [],
          "children": [],
          "content": null,
          "isSelfClosing": false
        },
        {
          "type": "Element",
          "span": {
            "start": 0,
            "end": 0
          },
          "tagName": "body",
          "namespace": "http://www.w3.org/1999/xhtml",
          "attributes": [],
          "children": [
            {
              "type": "Element",
              "span": {
                "start": 1,
                "end": 33
              },
              "tagName": "math",
              "namespace": "http://www.w3.org/1998/Math/MathML",
              "attributes": [],
              "children": [
                {
                  "type": "Element",
                  "span": {
                    "start": 7,
                    "end": 33
                  },
                  "tagName": "annotation-xml",
                  "namespace": "http://www.w3.org/1998/Math/MathML",
                  "attributes": [],
                  "children": [
                    {
                      "type": "Comment",
                      "span": {
                        "start": 23,
                        "end": 33
                      },
                      "data": "foo",
                      "raw": "<!--foo-->"
                    }
                  ],
                  "content": null,
                  "isSelfClosing": false
                }
              ],
              "content": null,
              "isSelfClosing": false
            }
          ],
          "content": null,
          "isSelfClosing": false
        }
      ],
      "content": null,
      "isSelfClosing": false
    }
  ]
}
