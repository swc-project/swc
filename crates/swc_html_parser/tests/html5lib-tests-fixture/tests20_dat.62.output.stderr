(1,6): expected-doctype-but-got-start-tag
(1,32): expected-closing-tag-but-got-eof